### DISCUSSION TITLE
UPDATE MODEL

### DISCUSSION THREAD

**Discussion by yelnamynhalej** [kyleskom/NBA-Machine-Learning-Sports-Betting#159] [Q&A]
I'm very new to data science and machine learning but I'm a very experienced bettor. I'm using this to learn so that I can create my own model one day. 

How do I use sqlite to see the updated data for the 2022-2023 season? I used the
 
python -m Get_Data
python -m Create_Games

python -m XGBoost_Model_ML
python -m XGBoost_Model_UO

but I'm not seeing the update team data or the updated dataset for this season in the directory

**Comment by ConnecticutHuskies** [comment]
I am inexperienced and this is one of my many questions. commenting for visibility

**Comment by tjones-commits** [comment]
you can use this to view the sqlite databases. https://sqlitebrowser.org/

I make a copy of the sqlite file and change the file extension to .db before I open it. That might be a config issue on my end but I'm not sure.
